tests/foreman/virtwho/api/test_nutanix.py

```diff
@@ -197,3 +197,104 @@ def test_positive_hypervisor_id_option(
+                debug=True,
+                org=default_org.label,
+            )
+        # Check the option "prism_central=true" should be set in etc/virt-who.d/virt-who.conf
+        config_file = get_configure_file(virtwho_config.id)
+        assert get_configure_option("prism_central", config_file) == 'true'
+        virt_who_instance = (
+            target_sat.api.VirtWhoConfig()
+            .search(query={'search': f'name={virtwho_config.name}'})[0]
+            .status
+        )
+        assert virt_who_instance == 'ok'
+        hosts = [
+            (
+                hypervisor_name,
+                f'product_id={settings.virtwho.sku.vdc_physical} and type=NORMAL',
+            ),
+            (
+                guest_name,
+                f'product_id={settings.virtwho.sku.vdc_physical} and type=STACK_DERIVED',
+            ),
+        ]
+        for hostname, sku in hosts:
+            host = target_sat.cli.Host.list({'search': hostname})[0]
+            subscriptions = target_sat.cli.Subscription.list(
+                {'organization': default_org.name, 'search': sku}
+            )
+            vdc_id = subscriptions[0]['id']
+            if 'type=STACK_DERIVED' in sku:
+                for item in subscriptions:
+                    if hypervisor_name.lower() in item['type']:
+                        vdc_id = item['id']
+                        break
+            target_sat.api.HostSubscription(host=host['id']).add_subscriptions(
+                data={'subscriptions': [{'id': vdc_id, 'quantity': 'Automatic'}]}
+            )
+            result = target_sat.api.Host().search(query={'search': hostname})[0].read_json()
+            assert result['subscription_status_label'] == 'Fully entitled'
+        virtwho_config.delete()
+        assert not target_sat.api.VirtWhoConfig().search(
+            query={'search': f"name={form_data['name']}"}
+        )
+
+    @pytest.mark.tier2
+    def test_positive_prism_central_prism_central_option(
+        self, default_org, form_data, virtwho_config, target_sat
+    ):
+        """Verify prism_flavor option by "PUT
+
+        /foreman_virt_who_configure/api/v2/configs/:id"
+
+        :id: 7f3b18c7-178c-4547-86ed-0e34772f755f
+
+        :expectedresults: prism_flavor option can be updated.
+
+        :CaseLevel: Integration
+
+        :CaseImportance: Medium
+        """
+        value = 'central'
+        virtwho_config.prism_flavor = value
+        virtwho_config.update(['prism_flavor'])
+        config_file = get_configure_file(virtwho_config.id)
+        command = get_configure_command(virtwho_config.id, default_org.name)
+        deploy_configure_by_command(command, form_data['hypervisor_type'], org=default_org.label)
+        assert get_configure_option("prism_central", config_file) == 'true'
+        virtwho_config.delete()
+        assert not target_sat.api.VirtWhoConfig().search(
+            query={'search': f"name={form_data['name']}"}
+        )
```

Comment:
why reverting this from fixture teardown, isn't this working in teardown?

Reply:
> why reverting this from fixture teardown, isn't this working in teardown?

@Gauravtalreja1 Have restore the fixture for teardown. Thanks for your reminder